.circleci/config.yml

```diff
@@ -134,9 +169,22 @@ commands:
       - run:
           name: "App: Detox build"
           command: |
+            export MY_APP_MODE=""
```

Comment:
I believe we need this flag to be `mocked` for detox build? @kpchoy

Reply:
yes we still need this flag to be mocked. I think this is causing our pipeline to fail